### ISSUE TITLE
Certicate reported as expired after having been renewed

### ISSUE THREAD

**Issue by siebrand** [pgassmann/puppet-letsencrypt_nginx#13]
Not sure if this is something that can or should be resolved in this module.

I ran into this issue today:
- certificate was renewed 30 days ago.
- certificate was reported as expired today.

This was caused by nginx not having reloaded its configuration or having been restarted after the certificate had been renewed. Should this module do something about this?

**Comment by pgassmann** [comment]
I ran into that issue myself. But I did not yet look into it. There is an option to reload nginx when the certificates are renewed, `cron_success_command` 

https://github.com/voxpupuli/puppet-letsencrypt

```
letsencrypt::certonly { 'foo':
  domains => ['foo.example.com', 'bar.example.com'],
  manage_cron => true,
  cron_before_command => 'service nginx stop',
  cron_success_command => '/bin/systemctl reload nginx.service',
  suppress_cron_output => true,
}
```

But perhaps that is not the perfect solution. There is a renew hook option in certbot. See following blog and comments:
https://www.guyrutenberg.com/2017/01/01/lets-encrypt-reload-nginx-after-renewing-certificates/
